src/transformers/modeling_tf_bert.py

```diff
@@ -648,24 +671,29 @@ def call(
             output_attentions,
             output_hidden_states,
             return_dict,
-            training=training,
+            training,
         )
 
         sequence_output = encoder_outputs[0]
         pooled_output = self.pooler(sequence_output)
+        output = (
```

Comment:
Would move this after the next test, since it does not need to be computed when not using `return_dict`.